api/src/test/java/org/pmiops/workbench/api/config/TestBigQueryConfig.java

```diff
@@ -0,0 +1,47 @@
+package org.pmiops.workbench.api.config;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.gson.Gson;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.UUID;
+
+@SpringBootApplication(exclude = LiquibaseAutoConfiguration.class)
+public class TestBigQueryConfig {
+
+    @Bean
+    public BigQuery bigQueryService() throws Exception {
+        GoogleCredentials credentials = ServiceAccountCredentials
+                .fromStream(new FileInputStream(new File("bq-test-key.json")));
+
+        return BigQueryOptions.newBuilder()
+                .setProjectId("all-of-us-workbench-test")
+                .setCredentials(credentials)
+                .build()
+                .getService();
+    }
+
+    @Bean
+    public WorkbenchConfig workbenchConfig() throws Exception {
+        ObjectMapper jackson = new ObjectMapper();
+        String dataSetId = UUID
```

Comment:
I would add some common prefix to this (e.g. "test_") so we know we're dealing with a test data set... we could at some point write a tool to delete all the test data sets to clean up orphans.

Reply:
Good idea.